### ISSUE TITLE
where to setup/config?

### ISSUE THREAD

**Issue by jtkeyva** [alexfinnarn/d8_filestack#14]
does this module work? is there a setup/config page? i don't see any options anywhere. any help appreciated. thanks

**Comment by alexfinnarn** [comment]
Not really. My next step was to do this: https://github.com/alexfinnarn/d8_filestack/issues/13

You should be able to add a button to the CKEditor toolbar just as you would normally do so. I think the file gets uploaded to FileStack and inserted into the WYSIWYG, but I wanted to also add file entity support and a way to select images from FileStack and insert them. 

I can maybe advise you if you want to work more on the module, but I haven't done any D8 work in a long time.

**Comment by jtkeyva** [comment]
Cool thanks for the info! Don't have time to work so I am going to give the Cloudinary Module a try. Looks like there's a stable module which should do most of what I need.
